src/front/app/components/challengeTopics/ChallengeTopics.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import ACTIONS from './challangeTopicsActions';
+import ChallengeTopic from './ChallengeTopic';
+
+let ChallengeTopics = ({ challengeTopics = [], getChallengeTopics }) => {
+    if (!challengeTopics.length) {
+        getChallengeTopics();
```

Comment:
I guess that challenge topics will be already populated, you should dispatch GET_CHALANGE_TOPICS action when user navigates to this route